rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/rmnode/RMNode.java

```diff
@@ -232,10 +232,30 @@
     boolean isConfiguring();
 
     /**
-     * @return true if the node is locked, false otherwise.
+     * Returns a boolean saying whether the node is locked or not.
+     *
+     * @return {@code true} if the node is locked, {@code false} otherwise.
      */
     boolean isLocked();
 
+    /**
+     * Returns the timestamp at which the node has been locked.
+     * <p>
```

Comment:
Yes, it is valid (Javadoc is based on an very old version of HTML). Look at the seventh bullet of the following web page:

http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#format